dialog/file_goxjs.go

```diff
@@ -0,0 +1,34 @@
+//go:build wasm || js
+// +build wasm js
+
+package dialog
+
+import (
+	"fyne.io/fyne/v2"
+)
+
+func (f *fileDialog) loadPlaces() []fyne.CanvasObject {
+	return nil
```

Comment:
Every one of these functions is a dummy placeholder - I think we need TODO items and maybe even a linked issue if this PR is to land without file support in the browser.

Reply:
Will work on that this afternoon. Pushing an updated PR, but more work for this is needed.